api/templates/design.go

```diff
@@ -374,8 +374,7 @@ var ProjectLockedErrorType = Type("ProjectLockedError", func() {
 		Example("The project is locked, another operation is in progress, please try again later.")
 	})
 	Attribute("retryAfter", String, "Indicates how long the user agent should wait before making a follow-up request.", func() {
-		Example("<http-date>")
-		Example("<delay-seconds>")
+		Example("<http-date>/<delay-seconds>")
```

Comment:
Neviem, preco to generuje 2x aj `example` aj `examples`, ale asi nema zmysel na tom palit cas, staci takyto priklad.